async_retry_test.go

```diff
@@ -124,9 +124,16 @@ func Test_asyncRetry_Do(t *testing.T) {
 			name: "Timeout set correctly for each try",
 			args: args{
 				f: func(ctx context.Context) error {
+					started := time.Now()
```

Comment:
We want our test code to be as timing-independent as possible.
How about change option parameter to `Timeout(1 * time.Second)` and just add below before `select{}`.

```
select {
  case <-ctx.Done():
      // error
  default:
} 
```

Reply:
The approach you suggested would be more stable for testing. I fixed it!
https://github.com/Kyash/async-retry/pull/10/commits/daa4e7bd4711e3ef267157cde6bfc46cc4d5e87c